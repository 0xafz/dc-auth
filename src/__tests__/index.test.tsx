// import { render, screen } from "@testing-library/react"
import { render, screen } from "../test-utils"
import Home from "~/pages/index"

describe("Home", () => {
  it("renders a heading", () => {
    render(<Home />)

    const heading = screen.getByRole("heading", {
      name: /Join thousands of learners from around the world/i,
    })

    expect(heading).toMatchSnapshot()
    expect(heading).toBeInTheDocument()
  })
})
