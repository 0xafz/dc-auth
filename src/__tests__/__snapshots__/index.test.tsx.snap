// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders a heading 1`] = `
<h1>
  Join thousands of learners from around the world
</h1>
`;
